README.md

```diff
@@ -1,16 +1,18 @@
 # Reco
 
-Reco is an audio recorder focused on recording.
+Reco is a fully-featured audio recorder.
```

Comment:
Hmm, I think the key point of Reco is not that fully-featured, but that it focuses on recording rather than listening to recorded files—which means you can't listen to the files you recorded within Reco (you need to use some audio player) but instead you can select sound sources or file formats in the main window, not in the preference dialog.

Reply:
To me, "audio recorder focused on recording" seemed redundant since it was a noun and its verb form in the same sentence only a few words apart. I can't think of another word for "recording" that would be concise enough while still properly describing the app.

Reply:
Maybe something like "Reco is an audio recorder focused on being concise and simple to use."?

Reply:
> Maybe something like "Reco is an audio recorder focused on being concise and simple to use."?

That sounds nice for me.